website/aws.erb

```diff
@@ -876,6 +876,12 @@
                                 <li>
                                     <a href="/docs/providers/aws/r/dx_public_virtual_interface.html">aws_dx_public_virtual_interface</a>
                                 </li>
+                                <li>
+                                    <a href="/docs/providers/aws/r/dx_hosted_transit_virtual_interface.html">aws_dx_hosted_transit_virtual_interface</a>
```

Comment:
Nit: The sidebar should be alphabetically sorted

Reply:
Will do.